gating/pre_merge_test/run_system_tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+## Deploy virualenv for testing enironment molecule/ansible-playbook/infratest
+
+## Shell Opts ----------------------------------------------------------------
+
+set -ev
+set -o pipefail
+
+## Variables -----------------------------------------------------------------
+
+RE_HOOK_ARTIFACT_DIR="${RE_HOOK_ARTIFACT_DIR:-/tmp/artifacts}"
+SYS_WORKING_DIR=$(mktemp  -d -t system_test_workingdir.XXXXXXXX)
+export SYS_VENV_NAME="${SYS_VENV_NAME:-venv-molecule}"
+SYS_TEST_SOURCE_BASE="${SYS_TEST_SOURCE_BASE:-https://github.com/rcbops}"
+SYS_TEST_SOURCE="${SYS_TEST_SOURCE:-rpc-openstack-system-tests}"
+SYS_TEST_SOURCE_REPO="${SYS_TEST_SOURCE_BASE}/${SYS_TEST_SOURCE}"
+SYS_TEST_BRANCH="${SYS_TEST_BRANCH:-master}"
+export SYS_INVENTORY="/opt/openstack-ansible/playbooks/inventory"
```

Comment:
Is this the proper value for the dynamic inventory for an MNAIO deploy, or is there another value or variable that should be referenced?

Reply:
That's the proper location, but in a MNAIO deployment it won't be on the cloud host, it will be in the first infra VM IIRC. If you look into the details of https://github.com/rcbops/rpc-openstack/blob/master/gating/pre_merge_test/run_deploy_mnaio.sh you'll see that some prep is done on the host, and it's copied over to the infra vm, then executed there.

Reply:
@odyssey4me Since the location is the same in both cases, I will leave it as is here. I have added a commit to the executing script to us the `MNAIO_SSH` connection if it is set.

See commit https://github.com/rcbops/rpc-openstack-system-tests/pull/2/commits/6dcdd3a13c291b798af258a31db1e2ecba5b6504